### PR TITLE
added to_object method and added test for to_array as well.

### PR DESCRIPTION
**Pull request by sadams**

### CODE REVIEW

File: test/moment/to_array.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var moment = require('../../moment');
+
+exports.isValid = {
```
Comment (ichernev) [inline test/moment/to_array.js]: change to `exports.toArray = {`

**Comment by sadams** [comment]
@ichernev i have now updated the test names.

**Comment by ichernev** [comment]
@sadams can you please rebase on top of latest develop (directory structure was changed a bit in `2.10`). https://github.com/moment/moment/blob/develop/CONTRIBUTING.md

**Comment by ichernev** [comment]
Close in favor of #2506

**Pull request closed without merging**